### PR TITLE
Dev.update

### PR DESCRIPTION
**Pull request by tiberiu44**
This update solves breaking changes from numpy and torch and releases updated pre-trained models to work with the new torch versions.

### CODE REVIEW

File: stringlifier/modules/stringc2.py [hunk 1]
```diff
@@ -256,7 +256,7 @@ def _start_train(params):
     model.to(params.device)
     if params.resume:
         model.load('{0}.last'.format(params.output_base))
-    optimizer = torch.optim.Adam(model.parameters())
+    optimizer = torch.optim.Adam(model.parameters(), lr=1e-4)
```
Comment (Radu1999) [inline stringlifier/modules/stringc2.py]: I believe the learning rate should be specified in params, with a default value

**Review by Radu1999 — CHANGES_REQUESTED** [review]
